cypress/integration/common/actions.test.js

```diff
@@ -145,3 +146,68 @@ MultiSteps([
         searchValue,
     });
 });
+
+// WORKFLOW ACTIONS
+MultiSteps([
+    And,
+    Then,
+    When,
+], 'On {string} I click on {string} button for column with {string} value', (gridId, action, searchValue) => {
+    cy
+        .get(`[data-cy=${gridId}]`)
+        .should('be.visible')
+        .as('grid');
+    cy
+        .get('@grid')
+        .find('.workflow-designer-header-layer-cell')
```

Comment:
I am thinking if it's really necessary to do multiple finding elements. Gotta check what is really faster for cypress - single find, or multiple.